### PR TITLE
Processing comments in multiline statement

### PR DESCRIPTION
**Pull request by SlavaRa**

### CODE REVIEW

**Comment by SlavaRa** [comment]
@Neverbirth please review this PR

File: External/Plugins/ASCompletion/Completion/ASComplete.cs [hunk 1]
```diff
@@ -3528,7 +3528,7 @@ private static ASExpr GetExpression(ScintillaControl sci, int position, bool ign
                 // string literals only allowed in sub-expressions
                 else
                 {
-                    if (braceCount == 0) // not expected: something's wrong
+                    if (braceCount == 0 && !hadDot) // not expected: something's wrong
```
Comment (Neverbirth) [inline External/Plugins/ASCompletion/Completion/ASComplete.cs]: You sure this won't break anything out there? Sadly we don't have many tests for the GetExpression part yet.

Comment (SlavaRa) [inline External/Plugins/ASCompletion/Completion/ASComplete.cs]: Yes I sure

Comment (Neverbirth) [inline External/Plugins/ASCompletion/Completion/ASComplete.cs]: Well, obviously you wouldn't say no lol

Comment (SlavaRa) [inline External/Plugins/ASCompletion/Completion/ASComplete.cs]: yes off course ;)

**Pull request merged**